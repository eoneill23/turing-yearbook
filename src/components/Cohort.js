import React from 'react';
import Person from './Person';
import './Cohort.css';

function Cohort() {
  
}

export default Cohort;
