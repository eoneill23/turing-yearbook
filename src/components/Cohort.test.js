import React from 'react';
import { shallow } from 'enzyme';
import Cohort from './Cohort';

describe('Cohort', () => {
  it('should match the snapshot with all data passed in correctly', () => {
    const people = []
    const deleteStudent = jest.fn()
    const type = 'staff'
    const wrapper = shallow(<Cohort people={people} deleteStudent={deleteStudent} type={type}/>);
  
    expect(wrapper).toMatchSnapshot();
  });
});