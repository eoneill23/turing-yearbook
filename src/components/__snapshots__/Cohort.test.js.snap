// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cohort should match the snapshot with all data passed in correctly 1`] = `
<main
  className="cardsDisplay"
/>
`;
